src/main/java/org/jitsi/jigasi/JigasiBundleActivator.java

```diff
@@ -50,7 +53,8 @@
 
     public static BundleContext osgiContext;
 
-    private SipGateway gateway;
+    private SipGateway sipGateway;
+    private TranscriptionGateway transcriptionGateway;
```

Comment:
I feel a bit hesitant to add javadocs to stuff I did not write, I've added to the gateways but I don't know how to document osgiContect and UiserviceStub